lib/pf/Switch/Fortinet/FortiGate.pm

```diff
@@ -221,7 +221,7 @@ sub identifyConnectionType {
     my @require = qw(Fortinet-Vdom-Name);
     my @found = grep {exists $radius_request->{$_}} @require;
```

Comment:
I checked and  Fortinet-Vdom-Name is present in CLI login and VPN login. The only difference between CLI and VPN is Connect-Info. For CLI Connect-Info = admin-login, for VPN Connect-Info = vpn-ssl | vpn-ikev2